windows-agent/internal/proservices/landscape/executor.go

```diff
@@ -269,6 +269,15 @@ func installFromURL(ctx context.Context, homeDir string, downloadDir string, dis
 		}
 		return err
 	}
+	// If import was successful, let's launch cloud-init:
+	cmd := distro.Command(ctx, "cloud-init status --wait")
```

Comment:
This has to follow a similar abstraction done for the `touchdistro.Touch` in order to be testable with the mocks.